### DISCUSSION TITLE
Anyway to configure function file size limit

### DISCUSSION THREAD

**Discussion by JamesKai** [appwrite/appwrite#1658] [Q&A] [answered]
Hi, 

I have a function that is more than 10 MB (with many dependencies), but I could not deploy it to my Appwrite as the current Appwrite function is limited to 10 MB of file size. I would like to know if there is any way I could configure it, or if this limitation may be subject to change in the future. 

Thanks

**Comment by eldadfux** [comment, marked as answer]
You can change the max file limit in Appwrite by updating your setup environment variables. Specifically the `_APP_STORAGE_LIMIT ` variable. You can learn more here: https://appwrite.io/docs/environment-variables#storage
